### PR TITLE
Update prepare init-cluster message

### PR DESCRIPTION
**Pull request by kalensk**
Co-authored-by: Kalen Krempely <kkrempely@pivotal.io>
Co-authored-by: Jamie McAtamney <jmcatamney@pivotal.io>

### CODE REVIEW

File: cli/commanders/preparer.go [hunk 1]
```diff
@@ -71,7 +71,7 @@ func (p Preparer) InitCluster() error {
 		return err
 	}
 
-	gplog.Info("Gleaning the new cluster config")
+	gplog.Info("Attempting to initialize upgrade target cluster")
```
Comment (slari-pivotal) [inline cli/commanders/preparer.go]: How about something like:

"Starting target cluster initialization for upgrade"

**Comment by jchampio** [comment]
I thought we were keeping the old/new nomenclature for now?

**Comment by jmcatamney** [comment]
@slari-pivotal @jchampio `Attempting to initialize upgrade target cluster` is the message requested in the story, so we went with that.  We don't have strong feelings on the verbiage, and would be fine with changing it to `Starting new cluster initialization` or similar.

@skahler-pivotal Would you prefer we keep the message as specified or change it?

**Comment by skahler-pivotal** [comment]
Let's change it to fit what we had discussed and I think 

`Starting new cluster initialization`

is the better fit. We should also change the `status upgrade` message to match

`Initialize upgrade target cluster` -> `Initialize new cluster`

**Pull request merged**